Packs/FeedDshield/Integrations/FeedDshield/FeedDshield.py

```diff
@@ -0,0 +1,45 @@
+from CommonServerPython import *
+
+
+def main():
+    params = {k: v for k, v in demisto.params().items() if v is not None}
+
+    params['indicator_type'] = FeedIndicatorType.CIDR
+
+    params['url'] = 'https://www.dshield.org/block.txt'
+    params['ignore_regex'] = "[#S].*"
+    params['indicator'] = json.dumps({
+        "regex": r"^(\d{1,3}\.\d{1,3}\.\d{1,3}\.\d{1,3})\t[\d.]*\t(\d{1,2})",
+        "transform": "\\1/\\2"
+    })
+
+    fields = json.dumps({
+        "dshield_nattacks": {
+            "regex": "^.*\\t.*\\t[0-9]+\\t([0-9]+)",
+            "transform": "\\1"
+        },
+        # name will change.
```

Comment:
why

Reply:
@David-BMS - needed to decide on a name - eventually decided to take out the field altogether